src/OrchardCore.Themes/TheAdmin/Assets/scss/admin.scss

```diff
@@ -1,5 +1,10 @@
 @use 'sass:math';
 
+$zindex-sticky: 1020;
```

Comment:
Where this variable has been used?

Reply:
This is a bootstrap variable. I moved it into a global scope to ensure that bootstrap honers it when BS scss files are compiled.